pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -191,7 +191,7 @@ func (c *contract) sendAndWait(ctx context.Context, request *transaction.TxReque
 	}
 
 	if receipt.Status == 0 {
-		return txHash, transaction.ErrTransactionReverted
+		return txHash, c.txService.UnwrapRevertReason(ctx, request, transaction.ErrTransactionReverted)
```

Comment:
The process seems to fail before it would get a receipt. Supposedly the `EstimateGas` call (invoked by `c.txService.Send` above) returns with `exection reverted`.

Reply:
Fixed.